src/librustc/middle/stability.rs

```diff
@@ -543,11 +543,6 @@ pub fn check_pat(tcx: &ty::ctxt, pat: &hir::Pat,
         // Foo(a, b, c)
         hir::PatEnum(_, Some(ref pat_fields)) => {
             for (field, struct_field) in pat_fields.iter().zip(&v.fields) {
-                // a .. pattern is fine, but anything positional is
```

Comment:
Could we preserve this comment somehow? Mention on this that when the second field is None, it's a `..` pattern, which is okay.